decision-records/006-javascript-compatibility.md

```diff
@@ -0,0 +1,161 @@
+# JavaScript browser compatibility
+
+Date: 27/06/23
+
+Status: Accepted
+
+## Decision
+
+We will ship JavaScript that can parse without errors in browsers supporting `<script type="module">`.
```

Comment:
Sorry @romaricpascal I'm going to suggest we swap "ship" with "publish" consistently

```suggestion
We will publish JavaScript that can be parsed without errors in browsers supporting `<script type="module">`.
```